ajax/libs/video.js-chromecast/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "video.js-chromecast",
+  "version": "2.0.8",
+  "filename": "videojs-chromecast.min.js",
+  "description": "Chromecast plugin for videojs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/benjipott/video.js-chromecast.git"
+    },
+  "keywords": [
+    "chromecast",
+    "videojs",
+    "videojs-plugin"
+  ],
+  "author": "Benjipott, Inc.",
+  "license": "Apache-2.0",
+  "npmName": "videojs-chromecast",
```

Comment:
Let's use git auto-update because there are more versions from GitHub repository then npm. Hosting more versions are good for people to choose.
Let's modify from A to B as the followings:
- A
```js
  "npmName": "videojs-chromecast",
  "npmFileMap": [
    {
      "basePath": "",
      "files": [
        "*/"
      ]
    }
  ]
```
- B
```js
  "autoupdate": {
    "source": "git",
    "target": "https://github.com/benjipott/video.js-chromecast.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```